src/resolvers/index.ts

```diff
@@ -99,6 +101,13 @@ export function executeMutation<Args, ReturnType>(
     context: IContext
   ): Promise<ReturnType> {
     context.dbClient = writeClient();
+    Object.entries(args).forEach(([name, value]) => {
+      if (value instanceof Array && value.length > config.app.bulkSizeLimit) {
+        throw new UserInputError(
+          `Value of ${name} argument is too large (received=${value.length}, limit=${config.app.bulkSizeLimit}).`
```

Comment:
LGTM. 

here, we are limiting the input length. in case of bulk mutation, thinking if we should limit the `tags` associated with the savedItem. i guess its not needed coz our main concern with limiting to 30 is not to return a giant savedItem list as a response